src/unix/fs.c

```diff
@@ -965,18 +965,17 @@ static ssize_t uv__fs_sendfile(uv_fs_t* req) {
 
 #ifdef __linux__
     {
-      static int copy_file_range_support = 1;
+      static int no_copy_file_range_support;
 
-      if (copy_file_range_support) {
+      if (uv__load_relaxed(&no_copy_file_range_support) == 0) {
         r = uv__fs_copy_file_range(in_fd, &off, out_fd, NULL, req->bufsml[0].len, 0);
 
-        if (r == -1 && errno == ENOSYS) {
+        if (r == -1 && ((errno == ENOSYS) || (errno == EACCES && uv__is_buggy_cephfs(in_fd)))) {
```

Comment:
Can you retain the old layout? It was written that way to stay under the 80 column limit.

Reply:
@bnoordhuis Done, PTAL.